doc/cascadia/profiles.schema.json

```diff
@@ -327,6 +328,7 @@
         "sendInput",
         "setColorScheme",
         "setTabColor",
+        "selectAll",
```

Comment:
_Technically_, this should be its own PR and cherry-picked into the 1.14 branch... but the schema is online so I really don't see a reason for that.

Reply:
The schema is an aka.ms link pointed to the stable version's schema. If we move that forward to 1.14, the schema will report that selectAll is invalid...